### DISCUSSION TITLE
Share a grid view for edit use

### DISCUSSION THREAD

**Discussion by PhilDu77** [nocodb/nocodb#7876] [Features]
This issue has been already treated but I suppose now It is possible.
My use of NoCoDB was mostly for this need: share a grid view, only the grid (or a form)without the rest of function on DB, tables.
this give the possibility to users to add new records or acces a table from the DB.
Is there a way to obtain that, with special user profile or other way please ?

**Comment by PhilDu77** [comment]
I understand to do that in fact I have to create new user with editor right. after the user identification he can acces to the table with the right
to edit (add,remove some record).
Now I have the question to limit the edit or view to a selection of table...not for all

**Reply by donma033x** [reply]
I had the same need! Have you solve the problem now?
